gridview.jsx

```diff
@@ -1,3 +1,5 @@
+import React from 'react'
```

Comment:
Ezeket be kell tenni, különben nem épül be a bundle-ba, mert azt hiszi a webpack, hogy nincs rájuk referencia.